home.html

```diff
@@ -0,0 +1,275 @@
+    <img class="speaker-img" src="feature-img/lila.JPG" alt="">
+    <div>
+      <h3 class="speaker-name">
+        Lila Trektikov
+      </h3>
+      <p class="work">
+        Exercutive Director of the Wkimedia foundation
+      </p>
+      <br>
+      <p class="contribution">
+        Lila Tretkov is the Exercutive Director of the Wkimedia Foundation. 
+        The nonprofit organization that operates wikpedia. wkimedia is freely available in 209 languages and used to nearly half a billion people around the world every month
+      </p>
+    </div>
+  </div>
+  
+  <div class="sample">
+    <img class="speaker-img" src="feature-img/kilnam chon.JPG" alt="">
+    <div>
+      <h3 class="speaker-name">
+        Kilnam Chon
+      </h3>
+      <p class="work">
+        Kilnam Chon helped bring internet to Asia and is an outsspoken advocate for the open web commons
+      </p>
+      <br>
+      <p class="contribution">
+        In 2021, he was inducted into the inaugural class of the internet Society's (ISOC) internet Hall of Fame.
+      </p>
+    </div>
+  </div>
+  
+  <div class="sample">
+    <img class="speaker-img" src="feature-img/julia leda.jpg" alt="">
+    <div>
+      <h3 class="speaker-name">
+        Julia Leda 
+      </h3>
+      <p class="work">
+        President of young pirates of European
+      </p>
+      <br>
+      <p class="contribution">
+        European ingetration, political democracy and participation of youth through online as her major condem.
+        Reda's report outlining potential changes to EU copyright law approved by the parliament in July.
+      </p>
+    </div>
+  </div> -->
+  
+  <!-- <div class="sample">
+    <img class="speaker-img" src="feature-img/ryanmerkley.jpg" alt="">
+    <div>
+      <h3 class="speaker-name">
+        Ryan Merkley
+      </h3>
+      <p class="work">
+        CEO of creative Common, ex COO of the Mozila Foundation
+      </p>
+      <br>
+      <p class="contribution">
+        Ryan has been leading open source projects at the Mozila Foundation such source movement
+      </p>
+    </div>
+  </div>
+</div> -->
+<div class="contain">
+  <div class="partner">
+    <p>Partner</p> 
+  </div>
+  <div class="last-logo">
+    <div class="partner-logo">
+      <img src="logo/mozzilla.png" alt="">
+    </div>
+    <div class="partner-logo">
+      <img src="logo/naver.png" alt="">
+    </div>
+    <div class="partner-logo">
+      <img src="logo/airbnb.png" alt="">
+    </div>
+  </div>
+</div>
+<button class="morebtn">More</button>
+  <div class="copyright">
+  <div>
+  <img  class="last-content" src="logos/logo-cc.JPG" alt="">
+  </div>
+<div>
+  <p class=" last-content"> 
+  2015 creative  common korea some Rights Reserved 
+  European ingetration,  <br>political democracy and participation of youth through online <br>as her major condem.
+  Reda's  report outlining potential  <br> changes to EU  copyright law approved by the parliament in July.
+  </p>
+</div>
+  </div>
```

Comment:
- [ ] Kindly note that your `copyright` part is not visible on your website, kindly address that issue.